mkdocs.yml

```diff
@@ -237,7 +237,7 @@ nav:
           - Audit:
              - Access Audit Logs: setup_installation/admin/audit/audit-logs.md
              - Export Audit Logs: setup_installation/admin/audit/export-audit-logs.md
-  - <div class="dropdown"><button class="dropbtn"> API </button> <div id="myDropdown" class="dropdown-content"> <a id="hopsworks_api_link" href="https://docs.hopsworks.ai/hopsworks-api/dev">Hopsworks API</a> <a id="hsfs_api_link" href="https://docs.hopsworks.ai/feature-store-api/dev">Feature Store API</a> <a id="hsfs_javadoc_link" href="https://docs.hopsworks.ai/feature-store-api/dev/javadoc">Feature Store JavaDoc</a> <a id="hsml_api_link" href="https://docs.hopsworks.ai/machine-learning-api/dev">MLOps API</a> </div></div>: https://docs.hopsworks.ai
+  - <div class="dropdown"><button class="dropbtn"> API </button> <div id="myDropdown" class="dropdown-content"> <a id="hopsworks_api_link" href="https://docs.hopsworks.ai/hopsworks-api/dev">Hopsworks API</a> <a id="hsfs_api_link" href="https://docs.hopsworks.ai/feature-store-api/3.9.0-SNAPSHOT/">Feature Store API</a> <a id="hsml_api_link" href="https://docs.hopsworks.ai/machine-learning-api/">MLOps API</a> </div></div>: https://docs.hopsworks.ai
```

Comment:
Not really - the Feature Store API docuemntation and the MLOps API documentation don't exists anymore - they have been merged with the Hopsworks API, so those should go

And we should only have the java doc part which I don't know the link for tbh